gateway-ha/pom.xml

```diff
@@ -303,13 +303,6 @@
         </dependency>
 
         <!-- Test deps -->
-        <dependency>
```

Comment:
These are the old group id coordinates and org.wiremock is already declared as duplicate in the pom.